### DISCUSSION TITLE
誰買社群假帳號衝人氣？AI揪灌水網紅　準確率破九成

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#706] [General]
https://futurecity.cw.com.tw/article/1897
網路的世界越來越多的事情發生，光憑人類自己已無法辨別，只能借助ＡＩ　的力量

**Comment by Valentinaa0214** [comment]
社群上真的充斥著很多這種假帳號、幽靈帳號或是廣告帳號，有時候甚至會干擾到一般用戶使用社群平台的體驗，希望這項技術真的能有效的幫助社群公司官方管理這些帳號。
